ksql-engine/src/main/java/io/confluent/ksql/physical/QuerySchemas.java

```diff
@@ -38,11 +39,14 @@
 @Immutable
 public final class QuerySchemas {
 
+  private static final SqlSchemaFormatter FORMATTER =
+      new SqlSchemaFormatter(word -> false, Option.APPEND_NOT_NULL);
```

Comment:
why are we not using the ParserUtil::isResrevedIdentifier here?

Reply:
Because the old code didn't.  I've not changed functionality here.  As I called out in the description, there are places that don't pass reserved word.

This can be fixed, but requires all the expected topology files to be re-written, hence makes sense to do in another PR.